### PR TITLE
Fixed Spelling and Grammatical mistakes 

### PR DESCRIPTION
**Pull request by thisiskaransgit**
This PR fixes the grammatical and spelling errors found across the documentation.

Scope of the change : 
- typos
- missing commas, hyphens
- grammatical errors

This change will increase the readability of the documentation and will enhance the user experience.

### CODE REVIEW

**Comment by thisiskaransgit** [comment]
hey @GrahamTheDev, kindly review the PR

File: docs/how-does-daily-dev-work/community-picks-submission-guidelines.md [hunk 1]
```diff
@@ -44,9 +44,9 @@ Hopefully, the following guidelines will help you ensure that articles you scout
 
 Your own posts, promotional content, top 10s, listicles, vs articles, low-quality content, shallow or non-professional content, shortened URLs, redirected URLs or clickbait.
 
-- **Your own posts** - Community Picks is about highlighting amazing articles from our community, not self-promotion.
+- **Your own posts** - Community Picks are about highlighting amazing articles from our community, not self-promotion.
```
Comment (FrancescoXX) [inline docs/how-does-daily-dev-work/community-picks-submission-guidelines.md:47]: please revert this one

File: docs/how-does-daily-dev-work/community-picks-submission-guidelines.md [hunk 2]
```diff
@@ -44,9 +44,9 @@ Hopefully, the following guidelines will help you ensure that articles you scout
...
 - **Shortened URLs / Redirected URLs** - Please always submit the final and full URL (including the protocol e.g. https://mydomain.com/some-article). Any URLs submitted must not redirect or they will be rejected.
-- **Promotional content** - Community Picks are not meant for product and or Services promotion. If you want to promote your product or service, why not enquire about [advertising on daily.dev](https://promote.daily.dev/).
+- **Promotional content** - Community Picks are not meant for product and or Service promotion. If you want to promote your product or service, why not enquire about [advertising on daily.dev](https://promote.daily.dev/).
```
Comment (FrancescoXX) [inline docs/how-does-daily-dev-work/community-picks-submission-guidelines.md:49]: please revert this one

File: docs/how-does-daily-dev-work/reputation.md [hunk 3]
```diff
@@ -29,19 +29,18 @@ On 24th April 2022 we changed the way we calculated your reputation, the table b
 | You suggest a new source that is approved and added to the daily.dev feed! 🎉 | N / A | **200** |
 | Your comment gets featured (deprecated) | 2 | **N / A** |
 
- \* In order to gain or lose reputation from writing a post the post must be attributed to you and show in your posts list on your profile. Also please note that only upvotes from developers with more than 250 reputation points will count for the purpose of gaining reputation.  
+ \* In order to gain or lose reputation from writing a post, the post must be attributed to you and shown in your posts list on your profile. Also please note that only upvotes from developers with more than 250 reputation points will count for the purpose of gaining reputation.
```
Comment (FrancescoXX) [inline docs/how-does-daily-dev-work/reputation.md:32]: please revert this one

File: docs/how-does-daily-dev-work/reputation.md [hunk 4]
```diff
@@ -29,19 +29,18 @@ On 24th April 2022 we changed the way we calculated your reputation, the table b
 | You suggest a new source that is approved and added to the daily.dev feed! 🎉 | N / A | **200** |
 | Your comment gets featured (deprecated) | 2 | **N / A** |
 
- \* In order to gain or lose reputation from writing a post the post must be attributed to you and show in your posts list on your profile. Also please note that only upvotes from developers with more than 250 reputation points will count for the purpose of gaining reputation.  
+ \* In order to gain or lose reputation from writing a post, the post must be attributed to you and shown in your posts list on your profile. Also please note that only upvotes from developers with more than 250 reputation points will count for the purpose of gaining reputation.  
 
 ### What counts towards reputation?
-We introduced a system where you only receive reputation from an upvote on your post or comment if the person who **casts that upvote** has at least 250 reputation. 
+We introduced a system where you only receive a reputation from an upvote on your post or comment if the person who **casts that upvote** has at least 250 reputation.
```
Comment (FrancescoXX) [inline docs/how-does-daily-dev-work/reputation.md:35]: please revert this one

File: docs/key-features/community-picks.md [hunk 5]
```diff
@@ -16,7 +16,7 @@ To get access to this feature you first need to earn the Scout privilege (min. 2
 Community Picks are articles that are added to the daily.dev feed by our community members. These articles can be from nearly any website / online publication, not just from our 400+ sources.
 
 ### How can I spot a community pick?
-On a card in the feed there is always an icon showing the source in the top left.
+On a card in the feed, there is always an icon showing the source in the top left.
```
Comment (FrancescoXX) [inline docs/key-features/community-picks.md:19]: please revert this one

File: docs/key-features/community-picks.md [hunk 6]
```diff
@@ -52,7 +52,7 @@ If you do not have enough reputation to submit a link you will see a message tel
 ![](https://daily-now-res.cloudinary.com/image/upload/v1657719512/cp-4-not-enough-permissions_zadogk.jpg)
 
 ### How to submit a link
-On the menu on the left you will see a section "Contribute" containing a button "Submit article".
+On the menu on the left, you will see a section "Contribute" containing a button "Submit article".
```
Comment (FrancescoXX) [inline docs/key-features/community-picks.md:55]: please revert this one

File: docs/key-features/community-picks.md [hunk 7]
```diff
@@ -78,7 +78,7 @@ If the article already exists in the daily.dev feed you will see a message expla
 If the article you submitted has a valid URL and doesn't already exist you will get a confirmation that you will be notified by email about the request status.
 
 #### Email confirmation
-After a short while (up to an hour) you should receive an email telling you that your article submission was successful. In this email you will find a link to the article on daily.dev.
+After a short while (up to an hour) you should receive an email telling you that your article submission was successful. In this email, you will find a link to the article on daily.dev.
```
Comment (FrancescoXX) [inline docs/key-features/community-picks.md:81]: please revert this one

File: docs/key-features/community-picks.md [hunk 8]
```diff
@@ -106,7 +106,7 @@ There are multiple reasons why a submission to Community Picks could fail.
 ## Gaining Reputation
 Not only do you help the community by sharing useful and interesting articles, you can gain more reputation too!
 
-You gain reputation on any link you submit when it receives an upvote (as long as the person upvoting has at least 250 reputation), as well as the post author, if they have a profile on daily.dev.
+You gain reputation on any link you submit when it receives an upvote (as long as the person upvoting has at least 250 reputations), as well as the post author if they have a profile on daily.dev.
```
Comment (FrancescoXX) [inline docs/key-features/community-picks.md:109]: please revert this one

File: docs/key-features/the-companion.md [hunk 9]
```diff
@@ -15,7 +15,7 @@ Give the companion widget overview video a watch if you haven't already and see
 <iframe width="700" height="400" src="https://www.youtube.com/embed/7lfUzdkG03E" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 ## Activating the companion
-Getting started with the comanion widget is easy!
+Getting started with the companion widget is easy!
```
Comment (FrancescoXX) [inline docs/key-features/the-companion.md:18]: can you please Capitalize the c ?

File: docs/key-features/the-companion.md [hunk 10]
```diff
@@ -69,26 +69,26 @@ On the left of the companion widget you will see 5 buttons:
  1. You can switch to the writing tab at any point to create and edit your comment
  2. Enter your comment (we support markdown) into the text box
  3. You can also tag other people who use daily.dev in your comment using the @mention feature
- 4. Back at the top you can switch to the preview tab to make sure your comment looks awesome!
+ 4. Back at the top, you can switch to the preview tab to make sure your comment looks awesome!
```
Comment (FrancescoXX) [inline docs/key-features/the-companion.md:72]: please revert this one

File: docs/key-features/the-companion.md [hunk 11]
```diff
@@ -69,26 +69,26 @@ On the left of the companion widget you will see 5 buttons:
...
  5. Click on the "Comment" button to share your thoughts with the daily.dev community!
  
  The modal will automatically close once you have submitted your comment.
  
 ### 4. Bookmark
- Clicking on the bookmark button will add this article to your daily.dev bookmarks so you can easily find it later!
+ Clicking on the bookmark button will add this article to your daily.dev bookmarks so you can easily find them later!
```
Comment (FrancescoXX) [inline docs/key-features/the-companion.md:78]: please revert this one

File: docs/key-features/the-companion.md [hunk 12]
```diff
@@ -69,26 +69,26 @@ On the left of the companion widget you will see 5 buttons:
...
  
  Clicking on the button again will remove the article from your bookmarks.
  
 ### 5. Options
- Clicking on the options button will show a drop down with the following options:
+ Clicking on the options button will show a drop-down with the following options:
  1. **View discussion** - view the discussion on the daily.dev web app
- 2. **Share the article** - share the article via social media, email, WhatsApp etc.
+ 2. **Share the article** - share the article via social media, email, WhatsApp, etc.
```
Comment (FrancescoXX) [inline docs/key-features/the-companion.md:85]: please revert this one

File: docs/key-features/the-companion.md [hunk 13]
```diff
@@ -105,7 +105,7 @@ Your version should be at least v3.16.4)
 Please visit the [extension store](https://api.daily.dev/get?_ga=2.190141803.157894557.1655794404-623033389.1647175282) and download the latest version and try again.
 
 ### 2. Check your settings
-On the daily.dev extension go to the menu item "Customize".
+On the daily.dev extension, go to the menu item "Customize".
```
Comment (FrancescoXX) [inline docs/key-features/the-companion.md:108]: please revert this one

File: docs/key-features/the-companion.md [hunk 14]
```diff
@@ -119,7 +119,7 @@ If the above does not work, please let us know via [our report a bug form.](http
 Please remember to leave the best email to contact you on so we can help you fix the companion!
 
 ## Bugs!
-Hopefully you will never need to use this, but just in case a bug crept into the Companion please raise an issue using our [bug report template on GitHub](https://github.com/dailydotdev/daily/issues/new?assignees=&labels=Type%3A+Bug&template=---bug-report.md&title=%F0%9F%90%9B+BUG%3A+).
+Hopefully, you will never need to use this, but just in case a bug crept into the Companion please raise an issue using our [bug report template on GitHub](https://github.com/dailydotdev/daily/issues/new?assignees=&labels=Type%3A+Bug&template=---bug-report.md&title=%F0%9F%90%9B+BUG%3A+).
```
Comment (FrancescoXX) [inline docs/key-features/the-companion.md:122]: please revert this one

File: docs/setting-up-your-feed/blocking-tags-sources.md [hunk 15]
```diff
@@ -18,7 +18,7 @@ Blocking allows you to:
 2. Remove all articles that come from a [specific source](#blocking-sources) 
 
 :::info
-Blocking is only applied on articles that are displayed on the "**My Feed**" feed
+Blocking is only applied to articles that are displayed on the "**My Feed**" feed
```
Comment (FrancescoXX) [inline docs/setting-up-your-feed/blocking-tags-sources.md:21]: please revert this one

File: src/pages/faqs.md [hunk 16]
```diff
@@ -11,7 +11,7 @@ hide_table_of_contents: true
             <li>The first is that we source it from one of the 600+ sites that we gather articles from.</li>
         <li>The second is via submissions from the community via <a href="/docs/key-features/community-picks">Community Picks.</a></li>
         </ul>
-        <p>We therefore recommend you publish your articles on well known developer related publications.</p>
+        <p>We, therefore, recommend you publish your articles on well-known developer related publications.</p>
```
Comment (FrancescoXX) [inline src/pages/faqs.md:14]: please revert this one

**Review by FrancescoXX — COMMENTED** [review]
Thanks for this

File: docs/key-features/the-companion.md (hunk 9 above)
Comment (thisiskaransgit) [reply docs/key-features/the-companion.md]: Done !

**Comment by thisiskaransgit** [comment]
Hey @FrancescoXX @GrahamTheDev, I have implemented all the changes that have been asked, It's been a long time and the PR is still waiting to be merged, kindly take a look at it as the typos are present and visible on the documentation side, and will affect the user experience.

**Comment by thisiskaransgit** [comment]
Oh sorry for the removal of request for review, it happened by mistake 😢

**Comment by FrancescoXX** [comment]
hey @thisiskaransgit thanks for the PR. can you please open another PR with the changes or solve the conflicts and make a new one? Thank you

**Comment by FrancescoXX** [comment]
please update the pr to solve the conflicts, thnak you!

**Pull request closed without merging**